pkg/continuoustest/metrics.go

```diff
@@ -30,6 +32,12 @@ func NewTestMetrics(testName string, reg prometheus.Registerer) *TestMetrics {
 			Help:        "Total number of failed write requests.",
 			ConstLabels: map[string]string{"test": testName},
 		}, []string{"status_code", "type"}),
+		writesLatency: promauto.With(reg).NewHistogramVec(prometheus.HistogramOpts{
+			Name:        "mimir_continuous_test_writes_request_duration_seconds",
+			Help:        "Duration of the write requests.",
+			Buckets:     prometheus.ExponentialBuckets(0.001, 4, 6),
+			ConstLabels: map[string]string{"test": testName},
+		}, []string{"type"}),
```

Comment:
@krajorama could you help us here, would we want all new histograms to be native-histograms?

Reply:
Hi, I did not implements native histograms are this feature is still experimental. But changes are minor and resolutions is better so if you prefer just let me know and i would implement this change.

Reply:
Yes, please. I think keeping the new metrics to be native histogram-ready, at least, doesn't increase the amount of metrics to [migrate][1] in the future.

[1]: https://grafana.com/docs/mimir/latest/send/native-histograms/#migrate-from-classic-histograms

Reply:
Hi, I have updated the code to use native-histograms.